artworks/models.py

```diff
@@ -1,3 +1,87 @@
 from django.db import models
-
+from art_web_app.models import CustomModel
 # Create your models here.
+from django.contrib.auth.models import User
```

Comment:
I wouldn't import it this way. The reason is because you might want to change the default User model from Django. Using either `auth.contrib.auth.get_user_model` function to get the model class or `settings.AUTH_USER_MODEL` would be way better.